lang/python314-devel/files/patch-target-mm-malloc.diff

```diff
@@ -0,0 +1,13 @@
+Backported from https://github.com/NixOS/nixpkgs/commit/093fcd1629c4c0c9bfc0d6c7a538a679e9d481ef
+--- Modules/_hacl/include/krml/internal/target.h
++++ Modules/_hacl/include/krml/internal/target.h
+@@ -133,6 +133,9 @@
+       defined(_MSC_VER) ||                                                     \
+       (defined(__MINGW32__) && defined(__MINGW64_VERSION_MAJOR)))
+ #    define KRML_ALIGNED_MALLOC(X, Y) _aligned_malloc(Y, X)
++#  elif defined(__APPLE__)
```

Comment:
Maybe rather than making the patch conditional, use the correct condition here and apply the patch unconditionally? Otherwise we keep getting ports breakages due to patches not being rebased when needed.